.github/actions/action-npm-build/action.yml

```
@@ -0,0 +1,56 @@
+    required: true
+    type: string
+    description: path where the package.json is located
+  run-lint:
+    required: false
+    default: 'true'
+    description: Controls the execution of the 'npm run lint' script
+  run-test:
+    required: false
+    default: 'true'
+    description: Controls the execution of the 'npm run test' script
+outputs: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)